rice-middleware/impl/src/main/java/edu/arizona/rice/kim/service/impl/UaUiDocumentServiceImpl.java

```diff
@@ -3,7 +3,7 @@
  * Copyright 2009 State of Arizona Board of Regents
  *
  */
-package edu.arizona.kim.service.impl;
+package edu.arizona.rice.kim.service;
```

Comment:
I'll pull UA 2.1.9 and see how it was used there. This service class was not added from my side for KFS7 UAF-259. I'll find out if there's spring configuration not migrated as initial upgrade to 2.5.19 rice?